ext/drmgr/drmgr.c

```diff
@@ -288,7 +288,7 @@ static void
 drmgr_bb_exit(void);
 
 /* Reserve space for emulation specific note values */
-static ptr_uint_t note_base_emul;
+static ptr_uint_t note_base_emul = 0;
```

Comment:
Static vars are guaranteed 0 so this does nothing.

Reply:
Removed. Before, I saw some explicit zero initialisations for some static vars (e.g. in drbbdup.c, instrument.c), so thought of being verbose here.